core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -3806,7 +3899,7 @@ ExWorkProcRetcode ExHbaseAccessSQRowsetTcb::work()
                      // another chance in case there is more input data.
                      // If parent doesn't input any more data on second (or
                      // later) chances, then process the request.
-                     if (numRetries_ == 2)    {
+                     if (numRetries_ == 3)    {
```

Comment:
It was 3 earlier and I changed it 2 later. Now I bumped it 3, because the executor scheduler needs to cycle through the last list at least twice before this task could get a entry its down queue